source/css/scss/app/sitewide-appeal.scss

```diff
@@ -1,7 +1,19 @@
 .sitewide-appeal-wrapper {
+	display: none;
```

Comment:
always hidden until Contentful content is populated and all active/date checks are verified.